### DISCUSSION TITLE
Installable File for Salesforce Inspector - (EXE File, DMG File)

### DISCUSSION THREAD

**Discussion by sumanth86** [tprouvot/Salesforce-Inspector-reloaded#456] [General]
@tprouvot  Do we have any installable file for Inspector Plugin? because we have a lot of VDI (Virtual Desktop Images) that our team works on,  we want to provide the installable to our IT Team, so that we can have this setup in all VDI Images. This will help the team to open virtual machine and launch this Salesforce Inspector.

**Comment by tprouvot** [comment]
Hi @sumanth86 
Not sure to understand the need here, is it to  make installation automatic for certain users ? If yes I think this should be handled on your [organization chrome configuration](https://support.google.com/chrome/a/answer/6306504?hl=en) 

Extension file is basically the addon folder of the repository but if you make this you won't have the automatic updates of the extension

**Reply by sumanth86** [reply]
Yeah we have team who only use virtual desktop and for that we need to pre-install this plugin in chrome as a part of virtual desktop image.
They don't have option to install anything after launching virtual desktop.

**Reply by tprouvot** [reply]
How often is the image updated? For example how is chrome updated? The installation from a plugin file would require to enable the developer mode on chrome, do they have this permission?

**Reply by tprouvot** [reply]
Have you checked this:
https://www.chromium.org/administrators/pre-installed-extensions/
